src/main/java/com/employeesService/EmployeesService/exception/EmployeeException.java

```diff
@@ -0,0 +1,23 @@
+package com.employeesService.EmployeesService.exception;
+
+public class EmployeeException {
```

Comment:
As of now we have two identical classes with different names, use one instead and remove those two, name it something like `ApiError`